### PR TITLE
Unisell intro and how it works

### PR DESCRIPTION
**Pull request by gpark31**

### CODE REVIEW

File: views/index.handlebars [hunk 1]
```diff
@@ -31,5 +31,32 @@
         <p>There are no items yet. Click "List New Item" to add one.</p>
     {{/if}}
 {{else}}
-    <p>This is the about page.</p>
+    <hr>
+    <p><strong>What is Unisell?</strong></p>
```
Comment (matttarantino) [inline views/index.handlebars]: I don't believe we are allowed to use strong tags. must be done in css

File: views/index.handlebars [hunk 2]
```diff
@@ -31,5 +31,32 @@
...
+    <p>Unisell is a used-item marketplace exclusive to university students. Verified university students can buy or sell used 
+    items.</p>
+    <hr>
+    <p><strong>Our goal</strong></p>
```
Comment (matttarantino) [inline views/index.handlebars]: Same here

File: views/index.handlebars [hunk 3]
```diff
@@ -31,5 +31,32 @@
...
+    <p>Unisell is designed to improve college communities and promote reuse of items by providing a platform where students 
+    can safely interact.</p>
+    <hr>
+    <p><strong>How does Unisell work?</strong></p>
```
Comment (matttarantino) [inline views/index.handlebars]: And here

File: views/index.handlebars (hunk 1 above)
Comment (matttarantino) [reply views/index.handlebars]: Maybe use a h3 instead?

File: views/index.handlebars [hunk 4]
```diff
@@ -31,5 +31,32 @@
         <p>There are no items yet. Click "List New Item" to add one.</p>
     {{/if}}
 {{else}}
-    <p>This is the about page.</p>
+    <hr>
```
Comment (matttarantino) [inline views/index.handlebars:34]: Do we need this hr? there is nothing else on the page

File: views/index.handlebars [hunk 5]
```diff
@@ -31,5 +31,32 @@
...
+            <li>Create a bid.</li>
+            <li>You will be notified if your bid is accepted.</li>
+        </ol>
+        <p>Selling items:</p>
+        <ol>
+            <li>Create a post with photos, keywords, descriptions, etc.</li>
+            <li>Accept a bid that you like.</li>
+            <li>Communicate with the buyer to decide when and where to meet.</li>
+        </ol>        
+        <p><strong>*Note that all transactions happen on campus.</strong></p>
```
Comment (matttarantino) [inline views/index.handlebars]: and here

File: views/index.handlebars [hunk 6]
```diff
@@ -31,5 +31,32 @@
...
+
+</p>
```
Comment (matttarantino) [inline views/index.handlebars:68]: @gpark31 we also need to list which universities are on unisell (like in a ul). Take a look at the auth pages to see how to grab universities

**Review by matttarantino — CHANGES_REQUESTED** [review]
Looks good. See few comments below

File: views/index.handlebars (hunk 4 above)
Comment (gpark31) [reply views/index.handlebars:34]: It's personal design preference to divide it by sections if you really don't like it I could get rid of it.

File: routes/index.js [hunk 7]
```diff
@@ -24,11 +25,16 @@ const constructorMethod = (app) => {
       universityName = university.name;
       itemsList = await items.getAllByUniversityId(user.universityId);
     }
+    else{
+      let universityCollection = await universities.getAll();
+      universityCollection.forEach(university => universityList.push(university.name));
```
Comment (matttarantino) [inline routes/index.js:30]: I think you can do `universityList = await universities.getAll();` and you won't need line 30.

**Review by matttarantino — APPROVED** [review]
LGTM

**Pull request merged**